FilmFestivalLoader/IFFR/parse_iffr_html.py

```diff
@@ -478,15 +438,20 @@ def __init__(self, iffr_data, film):
         HtmlPageParser.__init__(self, iffr_data, 'CP')
```

Comment:
Please consider to change `'CP'` into a more adequate abbreviation, e.g. `'FI'`.

Reply:
Changed `'CP'` into `'FI'`, and, in the `ScreeningsPageParser`, `'F'` into `'FS'`.